test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -79,6 +79,16 @@ abstract class LorisIntegrationTest extends TestCase
         );
         $this->url = getenv('DOCKER_WEB_SERVER');
         $password  = new \Password($this->validPassword);
+        // generating a random JWTkey
+        $new_id = bin2hex(random_bytes(30)) . 'A1!';
+        $set    = [
+            'Value' => $new_id
+        ];
+        $where  = [
+            'ConfigID' => "66"
+        ];
+
+        $this->DB->update('Config', $set, $where);
```

Comment:
You'll need to select the original value first so that you can store it somewhere that you can restore it in the teardown. You should also be doing this by config name, not by a hardcoded config id